### ISSUE TITLE
[CST] Holistic design work for "Provide better 5103 notice request description"

### ISSUE THREAD

**Issue by skylerschain** [department-of-veterans-affairs/va.gov-team#81422]
## Value Statement
As a veteran receiving a 5103 Notice first person request in CST
I want to understand what that request means and what I need to do
So that I have access to evidence request information, can make an informed decision about whether I should waive my right to submit additional evidence, and have a direct on ramp to submit a 5103 waiver if I choose to do so

---
## Background Context

Current UI for 5103 Notice Response 1P Alert 👇

<img width="1307" alt="Screenshot 2024-05-06 at 3 50 24 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/95102058/9ec1acd6-8277-43f3-a68d-5f5a60e13dd1">

Current UI for "Ask for your claim decision" alert 👇

<img width="1319" alt="Screenshot 2024-05-06 at 3 50 28 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/95102058/5645752f-ea44-421b-b09f-584f1f35d80c">

There are a few different aspects of this experience that need to be considered holistically: 
- The 5103 1P alert (yellow) that shows up in the Status/Files tabs
- The 5103 1P alert "Document request" page where the Veteran can upload evidence
- The Claim Letters page with the "5103 notice" development letter (and waiver attached) that the Veteran can download 
- The "Ask for your claim decision" alert (blue)
- The "Ask for your claim decision" "Document request" page where the Veteran can do the same thing that the 5103 waiver does

Note: This is different from the [Claim Letters Navigation](https://github.com/department-of-veterans-affairs/va.gov-team/issues/81241) work, which established a consistent entrypoint to the Claim Letters page from CST.

## Design

WIP design work in Figma [here](https://www.figma.com/file/BUdMIxAiIutG12rZxZ0cg1/Claim-Letters-Navigation-and-5103-alert?type=design&node-id=1348-4779&mode=design&t=QfzlW8XszoSmpYcC-4).

Screenshot
<img width="1333" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBejJIQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--de2aacbee92010fe510341fa4afc471fbd8a5c5e/Screenshot%202024-04-23%20at%201.21.53%E2%80%AFPM.png" alt="Screenshot 2024-04-23 at 1.21.53 PM.png" />

### Enablement team (if needed)
@jstrothman 

## Engineering 
- Add here

## Out of scope
- Add here

## Open questions
- Add here

## Tasks
- [ ] Identify what 5103 1P alert language should be ([for this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/81396))
- [ ] Identify how 5103 Document Request flow should link to Claim Letters page so that Veterans can access the 5103 notice development letter
- [ ] Determine an optimal approach to the "Ask for your claim decision" alert given this 5103 1P request update

## Definition of Done
- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Design performs design QA and verifies the implementation matches the design spec
- [ ] Accessibility specialist performs accessibility review (in code or design)
- [ ] Engineering identifies staging users required to test and shares account and credentials with design and product
- [ ] Product performs functional QA and verifies acceptance criteria was met

## Acceptance Criteria
- [ ] _Different user flows are mapped out with mocks_
- [ ] _Recommendations for 5103 alert and Document Request page language are documented_
- [ ] _Initial recommendations for "Ask for your claim decision" banner are presented, to be fleshed out in [this epic](https://github.com/department-of-veterans-affairs/va.gov-team/issues/57104)_
- [ ] _The 'Request type:' in the Documents filed section is aligned with the name used on the evidence request alert_

**Comment by laurwill** [comment]
Hi @skylerschain , I did some explorations of a different way to present the 5103 notice and letter in [ this little content zone ](https://www.figma.com/file/BUdMIxAiIutG12rZxZ0cg1/Claim-Letters-Navigation-and-5103-alert?type=design&node-id=1601%3A14701&mode=design&t=o2t33vsbCX5LRXlt-1) you made for me. I'm trying to get the designs to better represent what the letter and waiver really are. But I think we still need more information before we can accurately design around this process.

**Does this letter duplicate information that Veterans may see in these other places?**
- [Evidence needed for your disability claim (static page)](https://www.va.gov/disability/how-to-file-claim/evidence-needed/)
- Information in the 526EZ online form itself (intro page, document upload screens, etc)
- A 5103 notice the 526EZ team is currently adding to the 526 online form that will link to a generic version of this letter
- Standalone evidence requests in the CST that ask Veterans for some of the same forms or documents the 5103 letter tells them to submit

**Other SME questions to answer (unless you all have answers to these!):**
- Can SMEs provide examples of each type of [5103 notice in this Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1714592225475/65ac7d9b478d2efcaa5a5adecea578415d1647b8?sender=u3038037989bec1f893349894)? Standard 5103, 5103 Duty to Assist, and Custom 5103? We need to see the full content we're sending to Veterans in order to align the online experience with the paper letters.
- Does every Veteran who submits a standard disability claim get a 5103 notice in the mail and in the CST? 
- Do Veterans who submit a [fully developed claim](https://www.va.gov/disability/how-to-file-claim/evidence-needed/fully-developed-claims/) ever get a 5103 notice, or does selecting FDC on the claim form replace the 5103 notice and waiver process?
- Are there any other direct communications to Veterans that duplicate the purpose of the 5103 notice (telling Veterans what types of evidence they need to submit)?
- Is there always new and relevant information in this letter—meaning information that we haven't already told Veterans elsewhere in the claim process? Or is the letter likely to repeat & duplicate information Veterans have already heard?
- How does the 5103 notice and waiver that we're providing in the CST relate to the generic 5103 notice in the 526EZ online form? 
- In what order does VA send 5103 notices and standalone evidence requests? Could a Veteran see the notice and submit the waiver, and then get an evidence request from VA? 
- What happens if a Veteran has an open 1p evidence request in the CST and a 5103 waiver alert? Does submitting the waiver affect the open evidence request? 
- If VA is gathering records on the Veteran's behalf and the Veteran submits the waiver, what happens?
- After a Veteran submits the waiver, how quickly are they likely to get a decision on their claim? (Asking because the "ask for decision" screen makes it sound like it will be immediate, and want to correct that)

**Design questions to answer:**
- Can we remove the "Ask for decision" screen entirely at the same time as we update the 5103 waiver screen? These are exact duplicates of each other, so letting them both exist at the same time seems confusing.
- If we can't remove the "Ask for decision" screen yet, can we rewrite it so it at least makes a connection to the 5103 screen, and move away from "ask for decision" as the wording? 
- If it's truly necessary for Veterans to read the 5103 notice in order to understand what types of evidence they need to submit, should we require them to download the letter before they can submit the waiver? My hunch is that this letter probably doesn't always have new and relevant information. But if this letter is telling them things they don't already know about evidence we need for their claim, we may want to make it a louder/pushier experience in the tool.
